projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -82,6 +101,37 @@ const BarChart: FC< BarChartProps > = ( {
 		[ chartOptions.tooltip ]
 	);
 
+	const renderPattern = useCallback(
```

Comment:
I'm wondering if we could refactor it to some kind of pattern generator, which combines the patterns (say n) and directions (say m) so that it'd be able have a lot more unique patterns (n x m).

Also it might be good to extract it to a hook?

Reply:
Only the Lines pattern supports orientation, but the other patterns do have other modifier props so we might be able to generate some variation